app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -337,8 +337,14 @@ private int getSelectedLogo() {
         }
 
         if (mUsePhoneProvider.isChecked()) {
+            // Use the params Bundle to provide a default phone number that will populate the
+            // phone number field in PhoneVerificationActivity if the number is known in advance.
+            Bundle params = new Bundle();
+            params.putString(AuthUI.PHONE_VERIFICATION_DEFAULT_PHONE_EXTRA, "+13099126574");
```

Comment:
This is a minor point, but I think putting a random phone number into the sample will cause confusion.  Many people will want to use this sample to test phone auth with their real number.

This type of example snippet belongs in the README.

Reply:
Moved to README, let me know if you don't think it's in an appropriate location.